packages/core/src/contracts/ownedAddressTCRWithAppeals.ts

```diff
@@ -63,6 +63,7 @@ export class OwnedAddressTCRWithAppeals extends BaseWrapper<OwnedAddressTCRWithA
     return this.instance
       .NewListingWhitelistedStream({}, { fromBlock })
       .map((e) => e.args.listingAddress)
+      .distinct()
```

Comment:
What if something gets delisted and listed again. Remove them leave `distinct()` to the library user in all cases as we can't know the use-case

Reply:
Another example is listening on only latest events of any kind and refresh UI. Distinct would disable refreshing

Reply:
The point of this function is to get the set of currently whitelisted listings. If something is delisted and listed again it will be returned correctly. But this observable doesn't care which event is responsible for the whitelist. I intend to create other functions that return variations of these event streams for other uses.

Reply:
oh you mean if something is delisted but listed again after we've already subscribed to the observable? that could be an issue. I'll remove distinct for now, but there's problems with it either way.